app/models/all_casa_admin.rb

```diff
@@ -4,6 +4,14 @@ class AllCasaAdmin < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :invitable, :recoverable, :validatable, :timeoutable, invite_for: 1.weeks
+
+  def casa_admin?
+    false
+  end
+
+  def supervisor?
+    false
+  end
```

Comment:
I don't understand how an AllCasaAdmin is allowed to interact with the app exactly... I don't think Policies take that into consideration... But I tested them as a user in the Policy spec.

My intuition is that they should be superusers/have access to everything, but they seem to be kept separate, so I decided to do this rather than make a lot of changes in ApplicationPolicy.

Reply:
All casa admins are closer to admin accounts than super users. They are mainly used to either create new casa orgs and create new admin accounts. They can also do feature flags and make patch notes.